### DISCUSSION TITLE
Elevation

### DISCUSSION THREAD

**Discussion by F5MVO** [dl9rdz/rdz_ttgo_sonde#202] [TTGO smartphone app]
Can you add elevation in livemap.html

Thanks for your good job.

**Comment by dl9rdz** [comment]
Could you elaborate, what exactly do you mean?
- Altitude (as sent by the sonde, usually in geoid height) is already shown.
- If you want height above sea level (instead of geoid height), you would need a earth gravity model. I have included that in the android app, but the whole model does not fit on the TTGO. Doing the translation for every update every second via some web service in the Internet is probably not a good idea. One would need to somehow download the EGM model in the JavaScript code from somewhere in the Intenet. egm1996 is about 2MB, okish. Could fit into local storage a web page is allowed to use. If you have really good arguments why this could be useful, maybe we can find someone to implement this...
- If you want height above ground, you need a terrain model database. SRTM3 or even better SRTM1.  That is available for free, but even a single 1°x1° tile of SRTM3 is probably already pushing the limits of web local storage...  I would suggest this is more appropriately handled in applications on more powerful devices. Or one could use some web service, but not sure if some free one exists. Google API is 5$/1000 requests.
